app/app.go

```diff
@@ -172,6 +172,10 @@ func newAppWithDriver(d fyne.Driver, clipboard fyne.Clipboard, id string) fyne.A
 	return newApp
 }
 
+func rootConfigDir() string {
+	return app.RootConfigDir()
```

Comment:
Should we not just call that function directly instead of indirectly through this function?